packages/codemods/src/codeMods/__tests__/utilities/importUtils.test.ts

```diff
@@ -0,0 +1,72 @@
+import { getImportsByPath, repathImport } from '../../utilities';
+import { Project } from 'ts-morph';
+
+const fileName = 'mockImports.tsx';
+const rootPath = 'office-ui-fabric-react';
+const buttonPath = `${rootPath}/lib/Button`;
+
+describe('Import Utilities test', () => {
+  let project: Project;
+
+  beforeEach(() => {
+    project = new Project();
+    project.addSourceFilesAtPaths(`${process.cwd()}/**/__tests__/mock/utils/*.tsx`);
+  });
+
+  it('can find import based on exact string match', () => {
+    const file = project.getSourceFileOrThrow(fileName);
+    const tag = getImportsByPath(file, buttonPath);
+    expect(tag).toBeTruthy();
+    expect(tag![0].getModuleSpecifierValue()).toEqual(buttonPath);
+  });
+
+  it('only finds exact matches', () => {
+    const file = project.getSourceFileOrThrow(fileName);
+    const imp = getImportsByPath(file, rootPath);
+    expect(imp.length).toEqual(0);
+  });
+
+  it('can find a single import based on a regex string match', () => {
+    const file = project.getSourceFileOrThrow(fileName);
+    const imp = getImportsByPath(file, /office\-ui\-fabric\-react.+Button/);
```

Comment:
I don't think the - absolutely has to be escaped in a regex, outside of a character class?